controllers/glanceapi_controller.go

```diff
@@ -665,15 +666,15 @@ func (r *GlanceAPIReconciler) generateServiceConfigMaps(
 	using the Client Key provisioned on the Ceph cluster
 	**/
 	if gb == "rbd" {
-		templateParameters["CephClusterFSID"] = instance.Spec.CephBackend.CephClusterFSID
-		templateParameters["CephClusterMonHosts"] = instance.Spec.CephBackend.CephClusterMonHosts
-		templateParameters["CephClientKey"] = instance.Spec.CephBackend.CephClientKey
+		templateParameters["CephClusterFSID"] = instance.Spec.CephBackend.ClusterFSID
+		templateParameters["CephClusterMonHosts"] = instance.Spec.CephBackend.ClusterMonHosts
+		templateParameters["CephClientKey"] = instance.Spec.CephBackend.ClientKey
 		// The pool we write in glance-api.conf
-		templateParameters["CephPool"] = glance.GetCephGlancePool(instance)
+		templateParameters["CephPool"], _ = ceph.GetPool(instance.Spec.CephBackend.Pools, instance.Name)
```

Comment:
Might be best to use `"glance"` instead of `instance.Name` here, because the user might name the CR something other than "glance".  Similar to what we're doing for the Cinder operator here: https://github.com/openstack-k8s-operators/cinder-operator/blob/00a7370b3c7453406ab24c9e965cce26819a7769/controllers/cinder_controller.go#L653

Reply:
Yeah, as per our previous conversation on the cinder-operator patch (which is basically similar to this one), if the user named the instance with a diff name, the call to `ceph.GetPool` would return an error. 
Considering that we plan to have a single default related to a service type, I guess for now we can just change the ceph.GetPool to: `ceph.GetPool(instance.Spec.CephBackend.Pools, "glance")`, until new defaults are available and provided by lib-common `storage/ceph` module.

Reply:
OK, I've switched `instance.Name` to `"glance"` as suggested.